velocity.js

```diff
@@ -1447,7 +1488,7 @@
 				getUnit: function(str, start) {
 					var unit = (str.substr(start || 0, 5).match(/^[a-z%]+/) || [])[0] || "";
 
-					if (unit && CSS.Lists.units.indexOf(unit) >= 0) {
+					if (unit && $.inArray(unit, CSS.Lists.units) >= 0) {
```

Comment:
Gah - just had a look, and $.inArray is jQuery only - we don't shim it - if you take these out I'll add a shim when I get home then do the release